models/DataObject/ClassDefinition/Data/CalculatedValue.php

```diff
@@ -510,7 +510,7 @@ public function getSetterCodeLocalizedfields($class)
         $code .= '* @param ' . $this->getPhpdocInputType() . ' $' . $key . "\n";
         $code .= '* @return $this' . "\n";
         $code .= '*/' . "\n";
-        $code .= 'public function set' . ucfirst($key) . '(' . '$' . $key . ', $language = null): static' . "\n";
+        $code .= 'public function set' . ucfirst($key) . '(' . '$' . $key . ', ?string $language = null): static' . "\n";
```

Comment:
Should we add also the missing parameter type here?
```suggestion
        $code .= 'public function set' . ucfirst($key) . '(' . $this->getParameterTypeDeclaration() . ' $' . $key . ', ?string $language = null): static' . "\n";
```

Reply:
I was thinking about doing another pr, but I guess it is ok to do it in the same pr, so thanks for the suggestion!